src/blocks/helpers/block-utility.js

```diff
@@ -443,3 +447,25 @@ export const buildGetSyncValue = ( name, attributes, defaultAttributes ) => {
 		return attributes?.[field];
 	};
 };
+
+/**
+ * Get the reusable block content by id.
+ *
+ * @param {string} id The id of the reusable block.
+ * @returns {BlockInstance[]|*[]} The reusable block content.
+ */
+export function pullReusableBlockContentById( id ) {
```

Comment:
This isn't needed anymore, I think we can remove this and its reference in the css-handler/index.js file.

Reply:
We may keep it; pretty handy to have it for working with reusable blocks. Exploring the API is not fun, and rediscovering the function is not great.

Reply:
Alright if you think it can be useful in future, you can keep it.